wiki/New-Workspace-Setup.md

```diff
@@ -73,10 +73,31 @@ The Drasil directory structure relies on symbolic links. Windows 10/11 has suppo
 
 <summary><h4>Mac</h4></summary>
 
-Mac already comes with a shell + good terminal, so you will only need to install `git`.
+Mac already comes with a shell + good terminal, so you should only need to install `git` and GNU `grep`. You might already have `grep` installed, but the one by default shipped with macOS lacks support for the perl dialect of the extended regular expression language (which we use).
+
+##### `git`
 
 If you're using a recent version of Mac (> Mavericks/10.9), please open up a terminal window (`Apple Logo+SPACE`, then search "Terminal"), and type in `git --version` into it, and press enter. A GUI installer will appear prompting you to install Git. Please follow the on-screen steps, leaving all options default. Once the installation is complete, you should be able to type `git --version` into your Terminal window again and receive a version code back instead of having the GUI pop up again. If you're using an older version of Mac, you may follow through the steps of installing Git using any of the [official repositories](https://git-scm.com/download/mac).
 
+##### GNU `grep`
+
+Assuming you use Homebrew, you can use the [brew formula](https://formulae.brew.sh/formula/grep), installing GNU `grep` via `brew install grep`. Once `grep` is installed, you should be able to run `grep --version` and see a message along the following format:
```

Comment:
Doesn't seem to work. `brew install grep` does seem to install stuff (in `/opt/homebrew/bin` which is in my PATH) but not `grep`. There are no shortages of greps:
```
acBook-Pro-8:code carette$ ls /opt/homebrew/bin/*grep*
/opt/homebrew/bin/gegrep	/opt/homebrew/bin/msggrep
/opt/homebrew/bin/gfgrep	/opt/homebrew/bin/pcre2grep
/opt/homebrew/bin/ggrep		/opt/homebrew/bin/xzegrep
/opt/homebrew/bin/lzegrep	/opt/homebrew/bin/xzfgrep
/opt/homebrew/bin/lzfgrep	/opt/homebrew/bin/xzgrep
/opt/homebrew/bin/lzgrep	/opt/homebrew/bin/zstdgrep
```
My `grep` is still ancient
```
MacBook-Pro-8:code carette$ grep --version
grep (BSD grep, GNU compatible) 2.6.0-FreeBSD
```